02-express-tutorial/answers/07-params-query.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express')
+const app = express()
+const { products } = require('./data')
+
+app.get('/', (req, res) => {
+  res.send('<h1> Home Page</h1><a href="/api/products">products</a>')
+})
+app.get('/api/products', (req, res) => {
+  const newProducts = products.map((product) => {
+    const { id, name, image } = product
+    return { id, name, image }
```

Comment:
What's the reason destructing & reconstructing the object instead of just `return product`?

There are good reasons (it's not a trick question)!